js/app.js

```diff
@@ -0,0 +1,50 @@
+ // wait until the document is ready
+
+ $(function() {
+
+  // when someone hits submit, run the fucntion showCity
+
+$("form").submit(showCity);
+
+
+function showCity(event) {
+  event.preventDefault();
+
+
+    var city = $("#city-type").val();
```

Comment:
@kelseycoughlin consider making the value from the input the same case as what you are checking against in your `conditional statement`